experiments/yearbook/compare_trigger_policies/pipeline_config.py

```diff
@@ -41,7 +41,7 @@ def gen_pipeline_config(name: str, trigger: TriggerConfig, eval_strategy: EvalSt
             ],
             optimization_criterion=OptimizationCriterion(name="CrossEntropyLoss"),
             checkpointing=CheckpointingConfig(activated=False),
-            selection_strategy=NewDataSelectionStrategy(
+            selection_strategy=NewDataSelectionConfig(
                 maximum_keys_in_memory=1000, storage_backend="database", limit=-1, reset_after_trigger=True
```

Comment:
Otherwise this config class has a confusing name with the selection strategy class itself.

Reply:
Hmm, I don't know what is the most Pydantic way here: I would have named this `NewDataSelectionStrategyConfig` since it's a config for the selection strategy. But at the same time, this is also the object that _defines_ what selection strategy we use + the according config. 

@robinholzi how would you name this in this case?

Reply:
I assume, there multiple strategies that allow selection? If so I would leep the `Strategy` infix. If there's no other flavor of selection the Config is just a semantic subunit of `TrainingConfig`,  I'd choose `NewDataSelectionConfig .

Reply:
Thanks Xianzhe!